modules/configuration/src/test/java/org/apache/ignite/internal/configuration/util/ConfigurationUtilTest.java

```diff
@@ -204,29 +204,94 @@ public void findSuccessfully() {
                 )
         );
 
+        NodeValue<Object> root = find(List.of(), parentNode, true);
+        assertNull(root.field());
```

Comment:
Can we assert that it's actual field that we expect? We can check its name and declaring class, for example. NotNull is not enough